### DISCUSSION TITLE
Default username and password not working

### DISCUSSION THREAD

**Discussion by mepstein** [netbox-community/netbox-docker#673] [Getting Help] [answered]
I've been busily (and happily) populating my netbox-docker instance for the last few days. I don't recall having changed the default username/password of admin/admin, however, it no longer works. The VM that contains the docker container is locked down so that only specific source IPs can access it, so I hadn't yet gotten around to changing the password. It baffles me that it isn't working.

Is there a way that I can change the password via the command line? I tried using `manage.py createsuperuser`to create a new superuser account, however, when I execute a shell within the docker image it can't seem to locate python.

**Comment by tobiasge** [comment, marked as answer]
You need to use the Python installation from the virtual environment:
`/opt/netbox/venv/bin/python /opt/netbox/netbox/manage.py`

**Reply by mepstein** [reply]
Perfect. Worked like a charm! Thank you so much!
